includes/class-affiliatewp-leaderboard.php

```diff
@@ -0,0 +1,366 @@
+<?php
+/**
+ * Core: Plugin Bootstrap
+ *
+ * @package     AffiliateWP Leaderboard
+ * @subpackage  Core
+ * @copyright   Copyright (c) 2021, Sandhills Development, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       1.1
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) exit;
+
+/**
+ * Main plugin bootstrap.
+ *
+ * @since 1.0.0
+ */
+final class AffiliateWP_Leaderboard {
+
+	/** Singleton *************************************************************/
+
+	/**
+	 * Holds the instance
+	 *
+	 * @since 1.0
+	 * @var AffiliateWP_Leaderboard The one true AffiliateWP_Leaderboard
+	 */
+	private static $instance;
+
+	/**
+	 * The version number of AffiliateWP
+	 *
+	 * @since 1.0
+	 * @var   string
+	 */
+	private static $version;
+
+	/**
+	 * Main plugin file.
+	 *
+	 * @since 1.0.0
```

Comment:
The `$file` property was introduced in 1.1.
```suggestion
	 * @since 1.1
```